xml/System.Runtime.CompilerServices/Unsafe.xml

```diff
@@ -123,11 +129,13 @@
         <Parameter Name="right" Type="T&amp;" RefType="ref" />
       </Parameters>
       <Docs>
-        <typeparam name="T">To be added.</typeparam>
-        <param name="left">To be added.</param>
-        <param name="right">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <typeparam name="T">The type of reference.</typeparam>
+        <param name="left">The first reference to compare.</param>
+        <param name="right">The second reference to compare.</param>
+        <summary>
+          Determines whether the specified references point to the same location.
+        </summary>
+        <returns><see langword="true" /> if <paramref name="left"/> and <paramref name="right"/> point to the same location; otherwise, <see langword=false" />.</returns>
```

Comment:
missing quote for false

Reply:
I'm laughing that you find these so quickly. I stared at that line for five minutes and I didn't see it.